src/shell/dotnet/Shell/Fdc3/PreloadFdc3.cs

```diff
@@ -1,13 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿namespace Shell.Fdc3;
 
-namespace Shell.Fdc3
+public static class PreloadFdc3
 {
-    public static class PreloadFdc3
+    public static readonly string Fdc3BundleResourceName = $@"Shell.bin.{GetEnvironment()}.net6._0_windows.fdc3-iife-bundle.js";
+
+    private static bool _isDebugMode = false;
+    private static string GetEnvironment()
     {
-        public static readonly string Fdc3BundleResourceName = @"Shell.fdc3-iife-bundle.js";
+#if DEBUG
+        _isDebugMode = true;
```

Comment:
Get methods should not have side effects. `_isDebugMode` is not used elsewhere, you can just put the return into conditional compilation blocks:
```cs
#if DEBUG
        return "Debug";
#else
        return "Release";
#endif
```

Reply:
removed, as it is unnecessary